{"name": "Test login failure for non-existent user", "status": "passed", "start": 1734703515544, "stop": 1734703515546, "uuid": "aab5dd38-e2d8-41f1-8688-90dc84979307", "historyId": "8e43a90cdc817ed06c77125200f37c4c", "testCaseId": "8e43a90cdc817ed06c77125200f37c4c", "fullName": "tests.couriers.test_courier_login#test_login_non_existent_user", "labels": [{"name": "feature", "value": "Courier Login"}, {"name": "story", "value": "Login Failure"}, {"name": "parentSuite", "value": "tests.couriers"}, {"name": "suite", "value": "test_courier_login"}, {"name": "host", "value": "KZALANB019"}, {"name": "thread", "value": "11352-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.couriers.test_courier_login"}]}